src/wp-includes/class-wp-scripts.php

```diff
@@ -869,17 +901,29 @@ private function get_dependents( $handle ) {
 	 *
 	 * @since 6.3.0
 	 *
-	 * @param string $strategy The strategy to check.
+	 * @param string|mixed $strategy The strategy to check.
 	 * @return bool True if $strategy is one of the delayed strategies, otherwise false.
 	 */
-	private function is_delayed_strategy( $strategy ) {
+	private function is_delayed_strategy( $strategy ): bool {
 		return in_array(
 			$strategy,
 			$this->delayed_strategies,
 			true
 		);
 	}
 
+	/**
+	 * Checks if the provided fetchpriority is valid.
+	 *
+	 * @since 6.9.0
+	 *
+	 * @param string|mixed $priority Fetch priority.
+	 * @return bool Whether valid fetchpriority.
+	 */
+	private function is_valid_fetchpriority( $priority ): bool {
+		return in_array( $priority, array( 'auto', 'low', 'high' ), true );
+	}
+
```

Comment:
Looks like this function is a duplicate of what we already define in `src/wp-includes/class-wp-script-modules.php`.

Is it possible to leverage that existing implementation here instead of re-defining it?

Reply:
Can we move it to `src/wp-includes/functions.php` and reuse it in both file?

Reply:
I don't think it's worth adding a global function for such trivial logic.